src/Backend/Modules/MediaLibrary/Domain/MediaItem/TypeDBALType.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace Backend\Modules\MediaLibrary\Domain\MediaItem;
+
+use Doctrine\DBAL\Platforms\AbstractPlatform;
+use Doctrine\DBAL\Types\Type as DoctrineDBALType;
+
+final class TypeDBALType extends DoctrineDBALType
```

Comment:
you can remove a lot of the boilerplate code here by extending from the StringType (also on your other dbal types)